Makefile.toml

```diff
@@ -0,0 +1,40 @@
+extend = ".cargo/Makefile.toml"
+
+[tasks.build]
+clear = true
+command = "cargo"
+args = [
+  "build",
+  "--manifest-path",
+  "rustfmt-core/Cargo.toml",
+  "--workspace",
+]
+
+[tasks.install]
+command = "cargo"
+args = [
+  "install",
+  "--path",
+  ".",
+  "--force",
+  "--locked",
+]
+
+[tasks.test]
+clear = true
+run_task = { name = ["test-bin", "test-lib"] }
+
+[tasks.test-bin]
+env = { "RUSTFMT_MAKE_MANIFEST_PATH" = "rustfmt-core/rustfmt-bin/Cargo.toml" }
+run_task = "test-subproject"
+
+[tasks.test-lib]
+env = { "RUSTFMT_MAKE_MANIFEST_PATH" = "rustfmt-core/rustfmt-lib/Cargo.toml" }
+run_task = "test-subproject"
+
+[tasks.test-subproject]
+condition = { env_set = ["RUSTFMT_MAKE_MANIFEST_PATH"] }
+script = [
+  "cargo test --manifest-path ${RUSTFMT_MAKE_MANIFEST_PATH}",
```

Comment:
seems the Windows jobs aren't substituting here though seems right to me based on a cursory reading of the cargo make docs

Reply:
Looks like we need to change `script_runner` on windows: https://github.com/sagiegurari/cargo-make/issues/62.